test/grouping.jl

```diff
@@ -1200,4 +1200,25 @@ end
     end
 end
 
+@testset "non standard cols arguments" begin
+    df = DataFrame(x1=[1,2,2], x2=[1,1,2], y=[1,2,3])
+    gdf = groupby(df, r"x")
+    @test groupvars(gdf) == [:x1, :x2]
+    @test groupindices(gdf) == [1,2,3]
+
+    gdf = groupby(df, Not(r"x"))
+    @test groupvars(gdf) == [:y]
+    @test groupindices(gdf) == [1,2,3]
+
+    gdf = groupby(df, [])
+    @test groupvars(gdf) == Symbol[]
```

Comment:
```suggestion
    @test groupvars(gdf) == []
```
(`Symbol` doesn't make any difference here.)